plugins/inputs/gtfs/gfts_test.go

```diff
@@ -0,0 +1,191 @@
+package gtfs_test
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	gtfsbindings "github.com/MobilityData/gtfs-realtime-bindings/golang/gtfs"
+	"github.com/golang/protobuf/proto"
+	"github.com/stretchr/testify/require"
+
+	"github.com/influxdata/telegraf/plugins/inputs/gtfs"
```

Comment:
Don't import this and rather put the test into the same package.

Reply:
See above.